### DISCUSSION TITLE
Only show selected names

### DISCUSSION THREAD

**Discussion by GilbertBckr** [graphia-app/graphia#42] [Q&A] [answered]
I am currently plotting a social media graph. I was wondering if it was possible to only show the names of a few selected nodes opposed to showing no names at all or all names at once.

**Comment by timangus** [comment, marked as answer]
The default is to only display the names of the selected nodes, which if I understand is what you want? Perhaps you've changed this setting? Have a look at *View->Show Node Text* and make sure *Selected* is active.

**Reply by GilbertBckr** [reply]
Thank you very much for you reply. Is there any way to make this selection permanent?

**Reply by timangus** [reply]
As in make the names always visible regardless of selection? No there isn't, sorry. You could maybe use the *Bookmarks* menu to create a selection that you can recall though. This won't show the names all the time, but it does make it easy to quickly reselect the nodes of interest, as and when required.

**Reply by GilbertBckr** [reply]
Thanks a lot that was what I was looking for
